backend/seed.ts

```diff
@@ -160,6 +231,56 @@ async function main() {
     },
   })
 
+  // Add comment to Jon's post
+  await db.post.update({
+    where: {
+      id: 2,
+    },
+    data: {
+      threads: {
+        create: [
+          {
+            startIndex: 15,
+            endIndex: 38,
+            highlightedContent: 'Les Pluies de Castamere',
+            comments: {
+              create: [
+                {
+                  body: 'You bastard!',
+                  author: {
+                    connect: { id: tywin.id },
+                  },
+                },
+                {
+                  body: 'HA HA!',
+                  author: {
+                    connect: { id: ned.id },
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    },
+  })
+
+  // Add image and connect to post 1
+  await db.image.create({
+    data: {
+      smallSize:
+        'https://res.cloudinary.com/journaly/image/upload/v1574187295/journaly/p5dxflsivmpwii9146rw.jpg',
+      largeSize:
+        'https://res.cloudinary.com/journaly/image/upload/v1574187295/journaly/p5dxflsivmpwii9146rw.jpg',
+      imageRole: 'HEADLINE',
+      Post: {
+        connect: {
+          id: 1,
+        },
+      },
+    },
+  })
```

Comment:
I had started making some updates to other code to try to accommodate actually showing these images in the app, but I ran into a few road blocks and it also falls perfectly into the UI updates PR I'm going to do next, so I decided to revert those changes for now and just keep this nice and specific to updating the seed data and will revisit the rest in that next PR